boilerplate/registrations.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * This class defines all the routes to be registered on this application server
```

Comment:
How can I add more routes? Maybe drop a link to the relevant documentation page in here as a comment.

Reply:
ok